### PR TITLE
Switch to Python 3

### PR DESCRIPTION
**Pull request by hkamran80**
* Print statements have been modified to fit Python 3's minimums.
* Switch from `%s` to `.format()`
* Add end-of-line switch to normal colors with `\x1B[0m`
* Cleanup strings and dictionaries

### CODE REVIEW

File: phoneinfoga.py [hunk 1]
```diff
@@ -369,84 +370,85 @@ def osintScan():
...
-    print code_result + 'http://www.scamcallfighters.com/search-phone-%s.html' % number
-
-    tmpNumAsk = raw_input(code_info + "Would you like to search for temporary number providers footprints ? (Y/n) ")
+    print((code_info + "Generating URL on scamcallfighters.com...\x1B[0m"))
+    print(code_result + 'http://www.scamcallfighters.com/search-phone-{}.html\x1B[0m'.format(number))
 
+    tmpNumAsk = input(code_info + "Would you like to search for temporary number providers footprints? (Y/n) \x1B[0m")
     if tmpNumAsk.lower() != 'n' and tmpNumAsk.lower() != 'no':
-        print(code_info + '---- Temporary number providers footprints ----')
+        print((code_info + '---- Temporary number providers footprints ---m\x1B[0m'))
```
Comment (sundowndev) [inline phoneinfoga.py]:
```suggestion
        print((code_info + '---- Temporary number providers footprints ---\x1B[0m'))
```

**Comment by sundowndev** [comment]
Using 2to3 could've been more effective https://docs.python.org/3.7/library/2to3.html

**Comment by sundowndev** [comment]
Finally I'm already preparing the migration to Python3 in the next release (see develop branch).

**Comment by hkamran80** [comment]
No problem! Happy to help! This is a great tool, and I intend to use this!

**Pull request closed without merging**